app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -16,10 +16,22 @@ def show
         rental
       end
 
+      def connect_to_new
```

Comment:
i will try to extract this to connection controller and make this properly restful